integration/kubernetes/cleanup_env.sh

```diff
@@ -6,6 +6,10 @@
 #
 # This script is used to reset the kubernetes cluster
 
+set -o errexit
```

Comment:
I'm wondering if we can just unset the `errexit`, so the script does not quit on any fails. wdyt?

Reply:
I'm fine with the cleanup script does not quit on any fails. ;). 
Then I will remove the second commit about `|| true`, but i'm gonna stick with the `if statement`, since i really don't want the exit status of this script `cleanup_bare_metal_env.sh` be non-zero. ;)